tests/bluetooth/bsim_bt/bsim_test_iso/prj.conf

```diff
@@ -18,4 +18,6 @@ CONFIG_BT_CTLR_SYNC_ISO=y
 CONFIG_BT_CTLR_ADV_ISO_PDU_LEN_MAX=251
 CONFIG_BT_CTLR_SYNC_ISO_PDU_LEN_MAX=251
 
+CONFIG_BT_CTLR_ISO_VENDOR_DATA_PATH=y
+
```

Comment:
Make a new prj_vendor_dp.conf file or some suitable name and let vendor path be a new test run in CI, see the `compile.sh` how LOW LAT variant of controller is tested using existing test application.